_Week8/Group4/simulate.do

```diff
@@ -0,0 +1,40 @@
+cap prog drop income_sim
+
+prog define income_sim, rclass
+
+clear
+
+syntax anything
```

Comment:
The do-file needs substantially better commenting -- there is not much here! The `README.md` file should serve as a narrative guide, while the do-file itself should have its own condensed waypoints for the reader.